src/libraries/System.Private.CoreLib/src/System/Globalization/ISOWeek.cs

```diff
@@ -34,6 +34,8 @@ public static int GetWeekOfYear(DateTime date)
             return week;
         }
 
+        public static int GetWeekOfYear(DateOnly date) => GetWeekOfYear(date.GetEquivalentDateTime());
```

Comment:
Any new public new API need to have the Xml triple-slashes docs on it. Please add them to all new APIs.